doc/source/ecosystem.rst

```diff
@@ -327,6 +327,26 @@ PyTables, h5py, and pymongo to move data between non pandas formats. Its graph
 based approach is also extensible by end users for custom formats that may be
 too specific for the core of odo.
 
+`Pandaral·lel <https://github.com/nalepae/pandarallel>`__
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Pandaral·lel provides a simple way to parallelize your pandas operations on all your CPUs by changing only one line of code.
+If also displays progress bars.
+
+.. code:: python
+
+    # import pandarallel
```

Comment:
these comments arent doing much

Reply:
I removed the 2 firsts comments and kept the two last ones.
Please tell me if I also should remove last ones (but I think they are useful).